js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -2287,26 +2287,26 @@ createClient=Create client
 inputTypeRows=Input rows
 eventTypes.IDENTITY_PROVIDER_FIRST_LOGIN.description=Identity provider first login
 usedMemory=Used memory
-validatePasswordPolicyHelp=Determines if Keycloak should validate the password with the realm password policy before updating it. For the case when user's password is saved in LDAP, some Keycloak password policies will not work (Not Recently Used, Expire Password, Hashing Iterations, Hashing Algorithm) due the fact that Keycloak does not have direct control over the password storage. It is needed to enable password policies at the LDAP server layer if you want to leverage those password policies.
+validatePasswordPolicyHelp=Determines if Keycloak should validate the password with the realm password policy before updating it. For the case when the user's password is saved in LDAP, some Keycloak password policies will not work (Not Recently Used, Expire Password, Hashing Iterations, Hashing Algorithm). This is because Keycloak does not have direct control over the password storage. It is necessary to enable password policies at the LDAP server layer if you want to leverage those password policies.
```

Comment:
```suggestion
validatePasswordPolicyHelp=Determines if Keycloak should validate the password with the realm password policy before updating it. When the user's password is saved in LDAP, some Keycloak password policies do not work such as Not Recently Used, Expire Password, Hashing Iterations, and Hashing Algorithm. This situation occurs because Keycloak does not have direct control over the password storage.  if you want to use those policies, enable password policies at the LDAP server layer.
```